recipe/meta.yaml

```diff
@@ -14,26 +13,22 @@ source:
     - GEOSSTRtreeTest.cpp.patch  # [win]
 
 build:
-  number: 0
+  number: 1
   features:
-    - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py>=35]
+    - vc{{ vc }}  # [win]
+  run_exports:
+    # pretty poor backcompat.  SO name changes each release.
+    #   https://abi-laboratory.pro/tracker/timeline/geos/
+    - {{ pin_subpackage('geos', max_pin='x.x.x') }}
```

Comment:
My understanding is that this will enforce a pinning down to the micro version number. (Which we already do for this package BTW.)

Reply:
yes, exactly